# This is a temporary hack, since wxGTK does not currently install this header.

--- a/src/MainApp.cpp
+++ b/src/MainApp.cpp	2025-06-02 13:07:27.000000000 +0800
@@ -30,14 +30,12 @@
 #include "Settings.h"
 #include "Utils.h"
 
-#include "wx/clipbrd.h"
+#include <wx/clipbrd.h>
 #include <wx/image.h>
 #include <wx/tooltip.h>
 #include <wx/stdpaths.h>
 #include <wx/utils.h>
 
-#include <wx/debugrpt.h>
-
 IMPLEMENT_APP( Guayadeque::guMainApp );
 
 namespace Guayadeque {
@@ -451,13 +449,7 @@
 // -------------------------------------------------------------------------------- //
 void guMainApp::OnFatalException()
 {
-    wxDebugReport Report;
-    wxDebugReportPreviewStd Preview;
-
-    Report.AddAll();
-
-    if( Preview.Show( Report ) )
-        Report.Process();
+    // FIXME: support this in wx
 }
 
 }
